packages/ra-ui-materialui/src/list/datagrid/DatagridRow.tsx

```diff
@@ -150,6 +188,7 @@ const DatagridRow: FC<DatagridRowProps> = React.forwardRef((props, ref) => {
                 style={style}
                 hover={hover}
                 onClick={handleClick}
+                onAuxClick={handleAuxClick}
```

Comment:
I don't think this event handle the CTRL + click combo. But this combo might not be universal cross OSs. Any thoughts?

Reply:
Doesn't work in Safari
https://caniuse.com/?search=auxclick

Reply:
The new IE :joy:

Reply:
Hehe. What should I do? Is this ok, or I should change the implementation?

Reply:
why don't you just modify the `DatagridRow::handleClick` method to detect if the ctrlKey is pressed, and switch from history.push() to window.open in that case?

Reply:
Ok

Reply:
I can't detect the button clicked using `onClick` handler, i have to use `onMosueDown`